apps/ledger-live-desktop/src/renderer/families/cardano/MemoValueField.jsx

```diff
@@ -25,13 +28,18 @@ const MemoValueField = ({
 
   const onMemoValueChange = useCallback(
     memo => {
+      track("button_clicked", {
```

Comment:
I assume product set these even names?
If not, perhaps something more meaningful would be better 😕

Reply:
it is set by "analytics" team :neutral_face: